docs/wiki/[User-Guide]-Module-Releases.md

```diff
@@ -0,0 +1,93 @@
+## Release strategy
+
+The Azure landing zones Terraform module is designed to be published and consumed through the Terraform Registry.
+This approach allows us to release updates in a controlled manner, helping customers to manage change effectively.
+
+This also allows customers to implement recommended practices for consuming modules using [versioning][module_versions], mitigating the risk of unexpected or unwanted changes.
+
+Releases are tagged based on [Semantic Versioning 2.0.0][sem_ver_2] and release notes will always outline the changes being introduced.
+In most cases, we will include the following information:
+
+- What's changed?
+- New features
+- Fixed issues
+- Breaking changes
+
+Whilst we try to follow the published [Semantic Versioning 2.0.0][sem_ver_2] guidelines, module releases are typically versioned according to the following:
+
+### Major releases
+
+Major versions are typically used when one or more of the following is true:
+
+- Adding significant functionality to the module (such as the addition of the Virtual WAN capability in release [v2.0.0][release_v2_0_0])
+- A significant change is needed to the customer code which requires careful consideration before being able to successfully run `terraform plan`
+- Existing resources will be recreated with direct customer impact (such as redeployment of networking resources)
+- Changes to `default` values which would result in unexpected changes to deployed resources
+- Addition of new input variables without a `default` value
```

Comment:
I think we could use minor as well here. Don't think this always warrants major release.